diff --git a/node_modules/@capacitor/push-notifications/android/build.gradle b/node_modules/@capacitor/push-notifications/android/build.gradle
index 82c513f..bbfb90c 100644
--- a/node_modules/@capacitor/push-notifications/android/build.gradle
+++ b/node_modules/@capacitor/push-notifications/android/build.gradle
@@ -46,7 +46,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
-    lintOptions {
+    lint {
         abortOnError = false
     }
     compileOptions {
