diff --git a/node_modules/@capacitor/local-notifications/android/build.gradle b/node_modules/@capacitor/local-notifications/android/build.gradle
index 79081a3..12ec110 100644
--- a/node_modules/@capacitor/local-notifications/android/build.gradle
+++ b/node_modules/@capacitor/local-notifications/android/build.gradle
@@ -45,7 +45,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
-    lintOptions {
+    lint {
         abortOnError = false
     }
     compileOptions {
